doc/source/uml/h5particles.txt

```diff
@@ -0,0 +1,64 @@
+@startuml
+class H5Particles as "H5Particles(Group)" {
+   _v_name : String = <name>
+   particles : Group = ParticleItems
+   bonds : Group = BondItems
+   data : Table = Data
+}
+
+class ParticleItems as "ParticleItems(Group)" {
+   _v_name : string = "particles"
+   items : Table = ParticleTable
+   data : Table = Data
+}
+
+class BondItems as "BondItems(Group)" {
+   _v_name : string = "bonds"
+   items : Table = BondsTable
+   data : Table = Data
+}
+
+class ParticleTable as "ParticleTable(Table)" {
+   _v_name : String = "Particles"
+   row : ParticleTableRow
+}
+
+class BondsTable as "BondsTable(Table)" {
+   _v_name : String = "Bonds"
+   row : BondsTableRow
+}
+
+class Data as "Data(Table)" {
+   _v_name: String = "Data"
+   row : DataRow
+}
+
+class ParticleTableRow as "Particle(Row)" {
+   [ uid ] : String[32]
+   [ coordinates ] : Float64[3]
+}
+
+class BondTableRow as "Bond(Row)" {
+   [ uid ] : String[32]
+   [ particles ] : Integer
+   [ n_particles ] : Integer = MAX_NUMBER_PARTICLES_IN_BOND
```

Comment:
I would probably have this as `0 <= Integer <= MAX_NUMBER_PARTICLES_IN_BOND` to show the range better